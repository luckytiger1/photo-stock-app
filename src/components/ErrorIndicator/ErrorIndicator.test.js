import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import ErrorIndicator from './ErrorIndicator';

it('<Favorites /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<ErrorIndicator />)).toMatchSnapshot();
});
