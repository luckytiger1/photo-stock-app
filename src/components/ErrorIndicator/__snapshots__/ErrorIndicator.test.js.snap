// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Favorites /> renders correctly 1`] = `
<div>
  Error
</div>
`;
